src/app/loadout-drawer/loadout-utils.ts

```diff
@@ -786,11 +786,11 @@ function isFashionPlug(defs: D2ManifestDefinitions, modHash: number) {
 export function getModsFromLoadout(
   defs: D2ManifestDefinitions | undefined,
   loadout: Loadout,
-  unlockedPlugs: Set<number>,
+  unlockedPlugs?: Set<number>,
 ) {
   const internalModHashes = loadout.parameters?.mods ?? [];
 
-  return resolveLoadoutModHashes(defs, internalModHashes, unlockedPlugs);
+  return resolveLoadoutModHashes(defs, internalModHashes, unlockedPlugs || new Set());
```

Comment:
@robojumper or @bhollis any qualms doing this? I don't think it matters if we know the names of the "cheaper" mods as last I remember they were named the same.

Reply:
I think that's fine.